daffodil-test/src/test/java/org/goodudfs/example/StringFunctions/StringFunctionsProvider.java

```diff
@@ -0,0 +1,25 @@
+package org.goodudfs.example.StringFunctions;
+
+import org.apache.daffodil.udf.*;
```

Comment:
I'd prefer an explicit import of UDFunctionProvider.

Reply:
Done